### DISCUSSION TITLE
Linux kernel 6.5 with ntfs v3 enabled

### DISCUSSION THREAD

**Discussion by DavidDah** [1000001101000/Debian_on_Buffalo#193] [Q&A]
Linux kernel 6.5 is released officially and it includes kernel support for ntfs v3 filesystem. In comparison with old NTFS-3G FUSE userspace driver performances are about 3x higher which is particularly important on older Terastation systems such is TS Pro v2 (TS-HTGL and TS-RHTGL).
Could someone compile kernel package file for this devices with ntfs enabled?

**Comment by 1000001101000** [comment]
I can't say I've ever had a reason to plug an NTFS formatted drive into a Terastation but can imagine a scenario or two where that might be desirable.

The script I use for building those kernels can be found here:
[build_kernel_armel.sh](https://github.com/1000001101000/Debian_on_Buffalo/blob/master/Tools/kernel_tools/build_kernel_armel.sh)

It might be as easy as adding "CONFIG_NTFS3_FS=y" and any other needed options to [custom_configs_armel](https://github.com/1000001101000/Debian_on_Buffalo/blob/master/Tools/kernel_tools/custom_configs_armel) then running the script against a 6.5 kernel tree.
